experimental/enable-metrics.yml

```diff
@@ -0,0 +1,46 @@
+- path: /instance_groups/name=bosh/properties/director/metrics_server?/enabled?
+  type: replace
+  value: true
+- path: /instance_groups/name=bosh/properties/director/metrics_server?/tls?/ca?
+  type: replace
+  value: ((metrics_server_tls.ca))
+- path: /instance_groups/name=bosh/properties/director/metrics_server?/tls?/certificate?
+  type: replace
+  value: ((metrics_server_tls.certificate))
+- path: /instance_groups/name=bosh/properties/director/metrics_server?/tls?/private_key?
+  type: replace
+  value: ((metrics_server_tls.private_key))
+
+# metrics_server ca
+- path: /variables/-
```

Comment:
When the name in known in advance, you might be interested in being idempotent by specifying `/variables/name=metrics_server_ca?` as a path. This way your ops file is resilient even when variables already exist.